packages/frontend-2/pages/server-management/pending-invitations.vue

```diff
@@ -0,0 +1,313 @@
+
+const openDeleteInvitationDialog = (item: ItemType) => {
+  if (isInvite(item)) {
+    inviteToModify.value = item
+    showDeleteInvitationDialog.value = true
+  }
+}
+
+const closeInvitationDeleteDialog = () => {
+  showDeleteInvitationDialog.value = false
+}
+
+const handleSearchChange = (newSearchString: string) => {
+  searchUpdateHandler(newSearchString)
+}
+
+const deleteConfirmed = async () => {
+  const inviteId = inviteToModify.value?.id
+  if (!inviteId) {
+    return
+  }
+
+  const result = await adminDeleteMutation(
+    {
+      inviteId
+    },
+    {
+      update: (cache, { data }) => {
+        if (data?.inviteDelete) {
+          // Remove invite from cache
+          cache.evict({
+            id: getCacheId('AdminUserListItem', inviteId)
+          })
+          // Update list in cache
+          updateCacheByFilter(
+            cache,
+            { query: { query: getInvites, variables: resultVariables.value } },
+            (data) => {
+              const newItems = data.admin.inviteList.items.filter(
+                (item) => item.id !== inviteId
+              )
+              return {
+                ...data,
+                admin: {
+                  ...data.admin,
+                  inviteList: {
+                    ...data.admin.inviteList,
+                    items: newItems,
+                    totalCount: Math.max(0, data.admin.inviteList.totalCount - 1)
+                  }
+                }
+              }
+            }
+          )
+        }
+      }
+    }
+  ).catch(convertThrowIntoFetchResult)
+
+  if (result?.data?.inviteDelete) {
+    closeInvitationDeleteDialog()
+    triggerNotification({
+      type: ToastNotificationType.Success,
+      title: 'Invitation deleted',
+      description: 'The invitation has been successfully deleted'
+    })
+  } else {
+    const errorMessage = getFirstErrorMessage(result?.errors)
+    triggerNotification({
+      type: ToastNotificationType.Danger,
+      title: 'Failed to delete invitation',
+      description: errorMessage
+    })
+  }
+}
+
+const resendInvitation = async (item: InviteItem) => {
+  const inviteId = item.id
+  if (!inviteId) return
+
+  const result = await resendInvitationMutation({ inviteId }).catch(
+    convertThrowIntoFetchResult
+  )
+
+  if (result?.data?.inviteResend) {
+    successfullyResentInvites.value.push(inviteId)
+    triggerNotification({
+      type: ToastNotificationType.Success,
+      title: 'Invitation Resent',
+      description: 'The invitation has been successfully resent'
+    })
+  } else {
+    const errorMessage = getFirstErrorMessage(result?.errors)
+    triggerNotification({
+      type: ToastNotificationType.Danger,
+      title: 'Failed to resend invitation',
+      description: errorMessage
+    })
+  }
+}
+
+const {
+  result: extraPagesResult,
+  fetchMore: fetchMorePages,
+  variables: resultVariables,
+  onResult
+} = useQuery(getInvites, () => ({
```

Comment:
https://github.com/specklesystems/speckle-server/pull/1764#discussion_r1294459319

validate this in other files as well